dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -79,3 +81,10 @@ def test_job_id_error_handling(Cluster):
             return_string = 'Job <12345> submited to <normal>.'
             cluster.job_id_regexp = r'(\d+)'
             cluster._job_id_from_submit_output(return_string)
+
+
+def test_log_directory():
```

Comment:
Nice pytest trick: you can use the `tmpdir` fixture. This gives you a unique folder on each run but pytest keeps it after the execution, allowing you to look at it in case something goes wrong. pytest keeps the last N (maybe 5) tmpdir folders IIRC. 

```py
def test_log_directory(tmpdir):

with PBSCluster(..., log_directory=tmpdir.strpath as cluster:
    ...
```